Automattic/jetpack
# Add new sharing's variables

Fixes #5184
#### Changes proposed in this Pull Request:

Adding new 3 variables for sharing purposes.
1. %post_id% for getting post id
2. %site_url% for getting root url
3. %post_slug% for getting slug
#### Testing instructions:
1. enable sharing modules.
2. go to settings > sharings.
3. click on "add new services"
4. insert the custom format like **%site_url%**/read_offline/**%post_slug%** to get something like http://example.com/read_offline/hello-world

Screenshot when adding new services. notice all new variable are listed there.
![screen shot 2016-10-24 at 9 39 33 pm](https://cloud.githubusercontent.com/assets/1949957/19647816/7a716b26-9a32-11e6-82d0-acef214cf6ec.png)

cc @jeherve
